activerecord/lib/active_record/log_subscriber.rb

```diff
@@ -84,6 +84,20 @@ def sql_color(sql)
       def logger
         ActiveRecord::Base.logger
       end
+
+      def debug(progname = nil, &block)
+        return unless super
+
+        backtrace = Rails.backtrace_cleaner.clean caller
+
+        relevant_caller_line = backtrace.detect do |caller_line|
+          !caller_line.include?('/initializers/')
```

Comment:
Never mind, `backtrace.first` just works. I don't know why my brain was derping on this one.